sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -187,20 +187,25 @@ def move_to_joint_value_target(self, joint_states, wait=True,
         self._move_group_commander.set_joint_value_target(joint_states_cpy)
         self._move_group_commander.go(wait=wait)
 
-    def plan_to_joint_value_target(self, joint_states, angle_degrees=False):
+    def plan_to_joint_value_target(self, joint_states, angle_degrees=False, custom_start_state=None):
         """
         Set target of the robot's links and plans.
         @param joint_states - dictionary with joint name and value. It can
         contain only joints values of which need to be changed.
         @param angle_degrees - are joint_states in degrees or not
+        @param custom_start_state - specify if plannig start state different
+        then the current state
```

Comment:
```suggestion
        than the current state
```
same for all other comments. Can it fit in one line with the previous?

Reply:
fixed in 8a85ebb